src/app/loadout-drawer/loadout-apply.ts

```diff
@@ -700,6 +703,27 @@ function applyLoadoutMods(
   };
 }
 
+function isAssigningToDefault(
+  item: DimItem,
+  assignment: {
+    socketIndex: number;
+    mod: PluggableInventoryItemDefinition;
+  }
+) {
+  const socket = item.sockets && getSocketByIndex(item.sockets, assignment.socketIndex);
+  if (!socket) {
+    warnLog(
+      'loadout mods',
+      'Why does socket',
+      assignment.socketIndex,
+      'not exist on',
+      item.name,
+      item.hash
```

Comment:
@nev-r I don't think this should be happening but I did see it once and then couldn't repro it again.

Reply:
no such thing as over-logging at this juncture